generate_additional_files.py

```diff
@@ -22,7 +22,7 @@
 _Keywords=Color;ICC;Profile;Calibrate;Printer;Display;
 """
 
-additionalfiles.generate(DOMAIN, PATH, "./panels/color/cinnamon-color-panel.desktop", prefix, _("Color"), _("Color management settings"), "")
+additionalfiles.generate(DOMAIN, PATH, "./panels/color/cinnamon-color-panel.desktop.in.in", prefix, _("Color"), _("Color management settings"), "")
```

Comment:
Why are they called .in.in if they are only processed once?

Reply:
Fixed, I based it on what was used in 5.2.x